crates/next-api/src/module_graph.rs

```diff
@@ -574,14 +570,33 @@ async fn validate_pages_css_imports(
         // the same as the app module. If it isn't we know it isn't a valid place to import global
         // css.
         if parent_module != app_module {
-            CssGlobalImportIssue::new(parent_module, module)
-                .resolved_cell()
-                .emit();
+            candidates.push(CssGlobalImportIssue::new(parent_module, module))
         }
 
         GraphTraversalAction::Continue
     })?;
 
+    candidates
+        .into_iter()
+        .map(async |issue| {
+            // We allow imports of global CSS files which are inside of `node_modules`.
+            if !issue
+                .module
+                .ident()
+                .path()
+                .await?
+                .path
+                .contains("/node_modules/")
+            {
+                // TODO this is suboptimal because it's celling in a loop with a somewhat arbitrary
+                // order
+                issue.resolved_cell().emit();
```

Comment:
do the filter in parallel and then sequentially `emit`

both `resolved_cell` and `emit` are synchronous and we don't really expect a lot of issues, so a sequential loop should be fine